client/src/graphql/client.js

```diff
@@ -0,0 +1,11 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client';
+
+const createApolloClient = () => {
+
+  return new ApolloClient({
+    uri: 'http://localhost:5000/graphql', // Replace with your backend GraphQL server URL
```

Comment:
Probably want an ENV variable eventually

Reply:
Added another commit to the PR for this.